lib/src/crawl_async.dart

```diff
@@ -34,7 +34,7 @@ final _empty = Future<Null>.value(null);
 /// have not completed. If the [edges] callback needs to be limited or throttled
 /// that must be done by wrapping it before calling [crawlAsync].
 Stream<V> crawlAsync<K, V>(Iterable<K> roots, FutureOr<V> Function(K) readNode,
```

Comment:
The documentation says that `readNode` is allowed to return `null`, which will ignore the key for the rest of the graph. `build_resolvers` is actively using that, so I think `readNode` should be:

```suggestion
Stream<V> crawlAsync<K, V>(Iterable<K> roots, FutureOr<V?> Function(K) readNode,
```

Reply:
ok